File: Question8/Question8.cs
using System;

namespace Solution
{
    class Solution
    {
        static int solution(int n, int m, int[] costs)
        {
            // Write solution here
            return 0;
        }

        static void Main(string[] args)
        {
            String[] firstLine = Console.ReadLine().Split(' ');
            String[] secondLine = Console.ReadLine().Split(' ');

            int n = int.Parse(firstLine[0]);
            int m = int.Parse(firstLine[1]);

            int[] costs = Array.ConvertAll(secondLine, sTemp => int.Parse(sTemp));

            int answer = solution(n, m, costs);
            Console.WriteLine(answer);
        }
    }
}

File: Question8/Question8.cpp
#include <iostream>

using namespace std;

int solution(int n, int m, vector<int> costs)
{
    // Write solution here
    return 0;
}

int main()
{
    int n, m;
    cin >> n >> m;

    vector<int> costs(m, 0);
    for (int i = 0; i < m; i++)
    {
        cin >> costs[i];
    }

    int answer = solution(n, m, costs);
    cout << answer << endl;

    return 0;
}
